packages/cli-repl/src/mongosh-repl.spec.ts

```diff
@@ -526,18 +527,69 @@ describe('MongoshNodeRepl', function () {
       ]) {
         context(mode, function () {
           let getHistory: () => string[];
+          let getAllHistoryItems: () => string[];
 
           beforeEach(function () {
             const { history } = mongoshRepl.runtimeState().repl as unknown as {
               history: string[];
             };
             getHistory = () =>
               history.filter((line) => !line.startsWith('prefill-'));
+            getAllHistoryItems = () => history;
             for (let i = 0; i < prefill; i++) {
               history.unshift(`prefill-${i}`);
             }
           });
 
+          describe('history() command', function () {
+            it('returns a formatted array of history', async function () {
+              output = '';
+              input.write('history()\n');
+              await waitEval(bus);
+              expect(output).includes(
+                formatOutput(
+                  {
+                    value: getAllHistoryItems()
+                      .slice(1, getAllHistoryItems().length)
+                      .reverse(),
+                  },
+                  { colors: true, maxArrayLength: Infinity }
+                )
+              );
+            });
+
+            it('works with array operations', async function () {
+              output = '';
+              input.write('history().slice(history().length-10).reverse()\n');
+              await waitEval(bus);
+              const history = getAllHistoryItems().slice(1).reverse();
+              expect(output).includes(
+                formatOutput(
+                  {
+                    value: history.slice(history.length - 10).reverse(),
+                  },
+                  { colors: false, maxArrayLength: Infinity }
+                )
+              );
+            });
+
+            it('works without ()', async function () {
```

Comment:
one con of `history` is that one can't do `history().length` etc operations on it so maybe it's confusing to have both. I think keeping it JS-like and not implementing i.e. `history -10` might be the way to go